Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM quay.io/condaforge/miniforge3
 
 # if you forked pandas, you can pass in your own GitHub username to use your fork
 # i.e. gh_username=myname
-ARG gh_username=pandas-dev
+ARG gh_username=WillAyd
```

Comment:
This is temporary. There is a gap with this Dockerfile currently where it clones the main repo, but obviously none of the CMake stuff is in main